### PR TITLE
feat: update external approval, manually trigger

### PR DESCRIPTION
**Pull request by RainbowDashy**
Update external approval docs:

- Users need to trigger sending external approvals in the UI workflow manually.
- External approvals are sent in the GitOps workflow automatically.

### CODE REVIEW

**Review by ecmadao — APPROVED** [review]

File: content/docs/administration/webhook-integration/external-approval.md [hunk 1]
```diff
@@ -97,3 +97,15 @@ Specifically, the issue assignee must have an **identical** email at Feishu to r
 
 1. Go to Bytebase Settings, click on IM Integration.
 1. Fill in Application ID and Secret from the previous section. Click on Enable and Create button.
+
+## How to Use
+
+### UI workflow
+
+The issue creator needs to click on the bell button near the assignee to send the Feishu approval.
```
Comment (tianzhou) [inline content/docs/administration/webhook-integration/external-approval.md:105]:
```suggestion
The issue creator needs to click on the bell button near the assignee to send the Feishu approval request.
```

File: content/docs/administration/webhook-integration/external-approval.md [hunk 2]
```diff
@@ -97,3 +97,15 @@ Specifically, the issue assignee must have an **identical** email at Feishu to r
...
+
+![issue detail page with the bell icon near the assignee highlighted](/static/docs/external-approval-feishu-bell.webp)
+
+### GitOps workflow
+
+The Feishu approval is sent automatically.
```
Comment (tianzhou) [inline content/docs/administration/webhook-integration/external-approval.md:111]: automatically when?

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/docs/administration/webhook-integration/external-approval.md [hunk 3]
```diff
@@ -97,3 +97,15 @@ Specifically, the issue assignee must have an **identical** email at Feishu to r
...
+
+### UI workflow
+
+The issue creator needs to click on the bell button near the assignee to send the Feishu approval request.
+
+![issue detail page with the bell icon near the assignee highlighted](/static/docs/external-approval-feishu-bell.webp)
+
+### GitOps workflow
+
+The Feishu approval is sent automatically.
```
Comment (RainbowDashy) [inline content/docs/administration/webhook-integration/external-approval.md:111]:
```suggestion
The Feishu approval is sent automatically when we arrive at a new stage or either of the assignee and the SQL statement has changed.
```

File: content/docs/administration/webhook-integration/external-approval.md [hunk 4]
```diff
@@ -97,3 +97,15 @@ Specifically, the issue assignee must have an **identical** email at Feishu to r
...
+
+### UI workflow
+
+The issue creator needs to click on the bell button near the assignee to send the Feishu approval request.
+
+![issue detail page with the bell icon near the assignee highlighted](/static/docs/external-approval-feishu-bell.webp)
+
+### GitOps workflow
+
+The Feishu approval is sent automatically when we arrive at a new stage or either of the assignee and the SQL statement has changed.
```
Comment (tianzhou) [inline content/docs/administration/webhook-integration/external-approval.md:111]:
```suggestion
The Feishu approval is sent automatically when either of the following happens:
- Arrive at a new stage.
- Assignee has changed.
- SQL statement has changed.
```

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/docs/administration/webhook-integration/external-approval.md (hunk 4 above)
Comment (tianzhou) [reply content/docs/administration/webhook-integration/external-approval.md:111]: Be clear, then shorten. Not the other way around.

File: content/docs/administration/webhook-integration/external-approval.md [hunk 5]
```diff
@@ -97,3 +97,18 @@ Specifically, the issue assignee must have an **identical** email at Feishu to r
...
+The issue creator needs to click on the bell button near the assignee to send the Feishu approval request.
+
+![issue detail page with the bell icon near the assignee highlighted](/static/docs/external-approval-feishu-bell.webp)
+
+### GitOps workflow
+
+The Feishu approval is sent automatically when either of the following happens:
+- Arrive at a new stage.
+- Assignee has changed.
+- SQL statement has changed.
```
Comment (RainbowDashy) [inline content/docs/administration/webhook-integration/external-approval.md:114]:
```suggestion
- Issue arrives at a new stage.
- Issue assignee has changed.
- Task SQL statement has changed.
```

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**